src/dotnet-new-gt/PackGlobalTool.targets

```diff
@@ -0,0 +1,23 @@
+<Project>
+  <Target Name="PackGlobalTool" Condition="'$(PackageType)' == 'DotnetTool'" BeforeTargets="GenerateNuspec" DependsOnTargets="Publish">
+    <PropertyGroup>
+      <NuspecProperties>
+        publishDir=$(PublishDir);
+        version=1;
+		
+		<!-- TODO: get these filled in as needed -->
+        <!--licenseUrl=$(PackageLicenseUrl);
+        projectUrl=$(PackageProjectUrl);
+        iconUrl=$(PackageIconUrl);
+        serviceable=$(Serviceable);
+        copyright=$(Copyright);
+        description=$(Description);
+        repositoryUrl=$(RepositoryUrl);-->
+		
+		targetFramework=$(TargetFramework);
+        
+        MicrosoftNETCorePlatformsPackageVersion=$(MicrosoftNETCorePlatformsPackageVersion);

```

Comment:
You can remove this now. It was a workaround for 2.1.300-preview1 that is fixed in preview2